.github/workflows/ci.yml

```diff
@@ -1,23 +1,38 @@
-name: GitHub Actions
-on: [push, pull_request]
+name: Build, test, and lint
+on:
+  pull_request:
+  push:
+    branches: [master]
+  release:
+    types: [published]
 jobs:
-
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-
-    - name: Set up Go 1.13
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.13
-      id: go
+    - name: Set up Go
+      uses: actions/setup-go@v2
 
     - name: Check out code
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
 
-    - name: Build, Test, Lint
-      run: |
-        export GOPATH=$HOME/go && export GOBIN=$(go env GOPATH)/bin && export PATH=$PATH:$GOPATH&& export PATH=$PATH:$GOBIN && mkdir -p $GOBIN
+    - name: Build and push Docker images
+      uses: docker/build-push-action@v1.1.0
+      if: ${{ github.event_name == 'push' || github.event_name == 'release' }}
```

Comment:
I think we actually need to create a release in order to test this, right?
(There is a draft release that we can add to for this.)

Reply:
Yes, but only a release off `master`. Does this work for you?

Reply:
How does this whole PR look like now? We just need to manually push the `cnabio/signy-in-toto-verifier:latest` once (cleanest solution going forward), and we should be good to go.